NativeUI/UIMenu.cs

```diff
@@ -1416,13 +1424,13 @@ public int CurrentSelection
         /// <summary>
         /// Returns the title object.
         /// </summary>
-        public UIResText Title { get; }
+        public NativeText Title { get; }
```

Comment:
This is also backwards incompatible. Other stuff I've seen:
* Subtitle
* BannerSprite 
* BannerRectangle
* SetBannerType in MenuPool.cs, both overloads.
* Photo in TabView.cs

Sorry if I'm breaking your balls with this stuff, I just don't want to break any existing mods.

Reply:
Why we don't just throw everything out of the window and rewrite everything in a 2.0 release?

Reply:
I don't really follow GTA5 modding anymore, but I'd say the amount of mods being released is much smaller than before, so not many people would benefit from an incompatible rewrite. So the amount of effort needed is not proportional to the rewards.

Probably the best move right now is make the old classes wrappers around the new ones and then using them in the public APIs. What do you think?

Reply:
Done as part of 268d6535e1e28841ab28aa2c81e4cd36509a7fe6 (I think)